vendor.conf

```diff
@@ -10,7 +10,7 @@ go.etcd.io/bbolt a0458a2b35708eef59eb5f620ceb3cd1c01a824d # v1.3.3
 google.golang.org/grpc 39e8a7b072a67ca2a75f57fa2e0d50995f5b22f6 # v1.23.1
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 golang.org/x/text 19e51611da83d6be54ddafce4a4af510cb3e9ea4
-golang.org/x/sys c990c680b611ac1aeb7d8f2af94a825f98d69720 https://github.com/golang/sys
+golang.org/x/sys 52ab431487773bc9dd1b0766228b1cf3944126bf https://github.com/golang/sys
```

Comment:
This matches the updated version in opencontainers/selinux; https://github.com/opencontainers/selinux/blob/v1.3.1/go.mod#L7